linera-execution/src/execution_state_actor.rs

```diff
@@ -89,6 +90,7 @@ where
         Ok((code, description))
     }
 
+    #[allow(dead_code)]
```

Comment:
It would be best if we could update this method with the contents of the closure in `handle_requests` instead of keeping the old code.